pushapkscript/docker.d/worker.yml

```diff
@@ -127,15 +134,35 @@ products:
           skip_check_multiple_locales: true
           skip_check_same_locales: true
           skip_checks_fennec: true
-          override_channel_model: "single_google_app"
-          app:
-            package_names:
-              - "org.mozilla.fenix"
-              - "org.mozilla.fenix.beta"
-              - "org.mozilla.fenix.nightly"
-            service_account: "dummy"
-            credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FENIX_DEP_PATH" }
-            certificate_alias: "fenix"
+          apps:
```

Comment:
We usually add `test $envvar` for variables within this file so that we fail fast in case we missed them. But I see in `init_worker.yml` that we don't have safeguards for any of the staging stuff so that's fine then 👍